src/x86/sse42.rs

```diff
@@ -15,11 +16,151 @@ pub const _SIDD_CMP_EQUAL_ORDERED: i8 = 0b00001100;
 
 pub const _SIDD_POSITIVE_POLARITY: i8 = 0b00000000;
 pub const _SIDD_NEGATIVE_POLARITY: i8 = 0b00010000;
+pub const _SIDD_MASKED_POSITIVE_POLARITY: i8 = 0b00100000;
 pub const _SIDD_MASKED_NEGATIVE_POLARITY: i8 = 0b00110000;
 
 pub const _SIDD_LEAST_SIGNIFICANT: i8 = 0b00000000;
 pub const _SIDD_MOST_SIGNIFICANT: i8 = 0b01000000;
 
+pub const _SIDD_BIT_MASK: i8 = 0b00000000;
+pub const _SIDD_UNIT_MASK: i8 = 0b01000000;
```

Comment:
We'll want to add docs to each of these constants, but I'm happy to punt on that for now!

Reply:
Added some short doc strings.